### PR TITLE
Fix: UI toggle refelect mcuboot trait

### PR DESCRIPTION
**Pull request by kylebonnici**
This PR is to address [NCP-4387](https://nordicsemi.atlassian.net/browse/NCP-4387)

### CODE REVIEW

**Comment by boundlesscalm** [comment]
Is there a need to make an explicitly deprecated feature work partially?

File: Changelog.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 4.3.1 - Unreleased
+
+### Fixed
+
+-   The **Enable MCUBoot** toggle reflects the MCIBoot trait of the connected
+    device
```
Comment (greg-fer) [inline Changelog.md:6]:
```suggestion
-   The **Enable MCUboot** toggle reflects the MCUboot trait of the connected
    device.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]
It's MCUboot, though I also always want to capitalize the B. :)

**Review by greg-fer — APPROVED** [review]

File: Changelog.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+## 4.3.1 - Unreleased
+
+### Fixed
+
+-   The **Enable MCUboot** toggle reflects the MCUboot trait of the connected
+    device.
```
Comment (KievDevel) [inline Changelog.md:6]: I wouldn't say this is a bug fix, but rather just a change in the behavior. From my perspective this toggle had a different goal and didn't have to reflect the state of the device (might be I'm wrong).

```suggestion
### Changed

-   The **Enable MCUboot** toggle reflects the MCUboot trait of the connected
    device.
```

Comment (kylebonnici) [reply Changelog.md:6]: I believe it should be fixed as this was the previous UI and this can be considered as a regression.

Comment (greg-fer) [reply Changelog.md:6]: Not my piece of cake, but Kyle's argument seems valid to me at this point of this feature's life cycle.

**Pull request merged**